addons/mrp/models/mrp_production.py

```diff
@@ -847,7 +847,7 @@ def _get_moves_finished_values(self):
         for production in self:
             if production.product_id in production.bom_id.byproduct_ids.mapped('product_id'):
                 raise UserError(_("You cannot have %s  as the finished product and in the Byproducts", self.product_id.name))
-            moves = [production._get_move_finished_values(production.product_id.id, production.product_qty, production.product_uom_id.id)]
+            moves += [production._get_move_finished_values(production.product_id.id, production.product_qty, production.product_uom_id.id)]
```

Comment:
I would use .append() because: 

1. It's done the same way as the byproduct
2. it's two times faster and @ryv-odoo likes when it's faster